extensions/qute/runtime/src/main/java/io/quarkus/qute/runtime/jsonobject/JsonObjectConfigurator.java

```diff
@@ -0,0 +1,17 @@
+package io.quarkus.qute.runtime.jsonobject;
+
+import jakarta.enterprise.event.Observes;
+
+import io.quarkus.qute.EngineBuilder;
+
+public class JsonObjectConfigurator {
```

Comment:
We could get rid of this class if you annotate the `JsonObjectValueResolver` with `@EngineConfiguration` and register the class with an `AdditionalBeanBuildItem`.